ocs_ci/ocs/resources/mcg.py

```diff
@@ -883,21 +888,25 @@ def _compare_cli_hashes():
     @property
     def status(self):
         """
-        Verify noobaa status output is clean without any errors
+        Verify the status of NooBaa, and its default backing store and bucket class
 
         Returns:
-            bool: return False if any of the non optional components of noobaa is not available
+            bool: return False if any of the above components of noobaa is not in READY state
 
         """
         # Get noobaa status
-        status = self.exec_mcg_cmd("status").stderr
-        for line in status.split("\n"):
-            if (
-                any(i in line for i in ["Not Found", "Waiting for phase ready ..."])
-                and "Optional" not in line
-            ):
-                logger.error(f"Error in noobaa status output- {line}")
-                # Workaround for https://github.com/red-hat-storage/ocs-ci/issues/3570
-                return True
-        logger.info("Verified: noobaa status does not contain any error.")
-        return True
+        get_noobaa = OCP(kind="noobaa", namespace=self.namespace).get(
+            resource_name="noobaa"
```

Comment:
please save `"noobaa"` as a constant in constants.py (`NOOBAA_RESOURCE_NAME = "noobaa"`)